brpc-java-core/src/main/java/com/baidu/brpc/protocol/AbstractRequest.java

```diff
@@ -59,6 +59,7 @@
     private String serviceTag;
     private Integer readTimeoutMillis;
     private Integer writeTimeoutMillis;
+    private boolean heartbeat;
```

Comment:
`heartbeat`seems not very clear to me.
Could this be replaced by `isAlive`?

Reply:
heartbeat is used to identify if the request is heartbeat, since Dubbo heartbeat request has different logics.